WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -10753,7 +10753,7 @@
 			path = Classes;
 			sourceTree = "<group>";
 		};
-		29B97314FDCFA39411CA2CEA = {
+		29B97314FDCFA39411CA2CEA /* CustomTemplate */ = {
```

Comment:
Xcode and CocoaPods keep changing this and the other lines unrelated to the package in this diff. I don't know why... One sets it one way, the other reverts.